docs/doc_v1.0/Build_documentation.md

```diff
@@ -276,12 +354,18 @@ At present, health is randomly sampled from a uniform distribution. It
 may later have some relation to age. A health score of 1 is assumed to
 be a very healthy individual, a score of 0 is a very frail individual.
 
-#### Compliance
+#### Isolation Compliance
```

Comment:
For all of the compliance values, it would be good to add where they are set. For instance, I think some of them are set in the population generator.java, if I'm not mistaken.